leistungsbeschreibung/README.md

```diff
@@ -1,57 +1,57 @@
 # fiskaltrust.Leistungsbeschreibung
 
-fiskaltrust ist ein Softwarehersteller und entwickelt für Kassenhersteller und Kassenhändler Compliance-as-a-Service und Revisionssichere-Daten-as-a-Service Produkte.
+Als Softwarehersteller entwickelt fiskaltrust Produkte für Kassenhersteller und Kassenhändler im Bereich Compliance-as-a-Service und Revisionssichere-Daten-as-a-Service .
 
 ## Compliance-as-a-Service
 
-Als Poscreator erhalten Sie Compliance-as-a-Service, indem Sie fiskaltrust in Ihr Produkt integrieren. Durch die Übermittlung jedes Beleges, jeder Transaktion oder jeder relevanten Aktion an die fiskaltrust-Middleware vor deren Abschluss wird diese Konformität als Service bereitgestellt. Die fiskaltrust-Middleware bietet eine stabile Schnittstelle zu verschiedenen Technologien für das vom Poscreator erstellte Possystem und bietet einen einfachen Berührungspunkt zwischen Possystem und Fiskaltrust-Middleware.
+Als Kassenhersteller erhalten Sie Compliance-as-a-Service, indem Sie die fiskaltrust.Middleware in Ihr Produkt integrieren. Durch die Übermittlung der Belegdaten, Transaktionen und anderer relevanten Daten an die fiskaltrust.Middleware wird Konformität als Service bereitgestellt. 
 
-![Überblick über die Funktionsweise von fiskaltrust](../product-service-description/compliance-as-a-service/media/overview-pos-ft-middleware.png)                                
-
-Diese Lösung macht die fiskaltrust-Middleware zu einem wesentlichen Bestandteil des Possystems und damit auch zu dessen Instanz zur Registrierkasse.
+Die fiskaltrust.Middleware stellt eine länderübergreifende Schnittstelle zur Verfügung, die über verschiedene Technologien (WCF, REST, gRPC, Serial-stream oder TCP-stream protocol) für Ihr Kassensystem erreichbar ist.
 
-Die fiskaltrust-Middleware verarbeitet einige der wichtigsten Datenfelder auf dem Beleg. Die Belegnummer als eindeutige Identifikation eines von der Registrierkasse übermittelten Beleges wird von der fiskaltrust-Middleware erstellt, um sicherzustellen, dass jeder Beleg von ihr verarbeitet wird.
+![Überblick über die Funktionsweise von fiskaltrust](../product-service-description/compliance-as-a-service/media/overview-pos-ft-middleware.png)                                
 
-Die Konformität wird durch die Kombination mehrerer Methoden und Komponenten erreicht.
+Die fiskaltrust.Middleware wird zu einem wesentlichen Bestandteil Ihres Kassensystems und gewährleistet damit die Konformität als Registrierkasse. Sie erzeugt die Belegnummer als eindeutige, fortlaufende Identifikation eines von der Registrierkasse übermittelten Beleges und stellt somit nachvollziebar dar, dass jeder Beleg von ihr verarbeitet wurde.
```

Comment:
"Konformität als Registrierkasse" --> "Konformität als Kassen- oder Aufzeichnungssystem" (Registrierkasse gibt es in DE nicht)

"eines von der Registrierkasse" --> "eines vom Kassen- oder Aufzeichnungssystem"

Es werden nicht nur "Belege" sondern auch "Transaktionen" gesichert und sollte daher ersetzt werden.
Man könnte anstelle dieser Begriffe aber auch "Request" (siehe unten) verwenden. 
Jedoch setzt sich diese nicht korrekte Bezeichnung als "Beleg" auch in der Doku fort: https://docs.fiskaltrust.cloud/doc/interface-doc/doc/

Reply:
Danke fürs Feedback, ich habe es in meinem aktuellen Commit berücksichtigt: https://github.com/christian-rogobete/productdescription-de-doc/commit/2805ee1b9398e0f0b6436d027e64f58da528cd86